src/Data/Tree/Binary.agda

```diff
@@ -0,0 +1,47 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Binary Trees
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Tree.Binary where
```

Comment:
Haha, I sense this is another awkward module organisation conversation coming up. Seems inconsistent to have `Data.AVL` but `Data.Tree.Binary`... Personally I'd be in favour of having `Data.Tree.AVL` but obviously that involves another round of deprecations.